test/parser/parser_test.cpp

```diff
@@ -65,6 +65,15 @@ TEST_F(ParserTests, BasicTest) {
       "CREATE TABLE students (name TEXT, student_number INTEGER, city TEXT, "
       "grade DOUBLE)");
 
+  // CREATE temporary table
```

Comment:
Is it better to write more C++ tests to test the functionalities apart from parser?

Reply:
We should have C++ tests and Java tests for this.